### ISSUE TITLE
QUEST: The Defias Brotherhood [quest=155]

### ISSUE THREAD

**Issue by MarrowB83** [chromiecraft/chromiecraft#1165]
#### WHAT CLIENT DO YOU PLAY ON?

- [ ] enGB
- [x] enUS
- [ ] other (specify)

##### FACTION

- [x] Alliance
- [ ] Horde

##### CONTENT PHASE:

- [ ] Generic
- [x] 1-19
- [ ] 20-29
- [ ] 30-39
- [ ] 40-49
- [ ] 50-59

##### CURRENT BEHAVIOUR:

I have accepted the quest, and it started properly. What is bugged here is NPC behaviour of "Defias Traitor". If he walks on the road and, for example, Goretusk attack us, he start spinning and spamming the same phrase into a chat and he wont help with attacking hostile monster.

I find out, that when the "Defias Traitor" reaches Moonbrook, he stops running, just walking, and he works completely fine.

##### EXPECTED BLIZZLIKE BEHAVIOUR:

https://www.youtube.com/watch?v=_XVhZfSkBfM

##### STEPS TO REPRODUCE THE PROBLEM:

1. Pickup the quest
2. Go on the road with Defias Traitor
3. Try attacking some Goretusk

##### AC HASH/COMMIT:

https://github.com/chromiecraft/azerothcore-wotlk/commit/7c21d567d59153b37978756a5bf02daf6998e665

##### OPERATING SYSTEM:

Ubuntu 20.04

##### MODULES:

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-CarbonCopy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-LevelUpReward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-Zonecheck](https://github.com/55Honey/acore_Zonecheck)

##### OTHER CUSTOMIZATIONS:

None.

##### SERVER:

ChromieCraft

**Comment by jamadaha** [comment]
Tried following the steps, found that when the Goretusk had me aggroed The Defias Traitor ran, sometimes spinning, while constantly yelling.
![image](https://user-images.githubusercontent.com/53914190/125197926-faa7bf00-e25f-11eb-8602-e16741f27e41.png)

**Comment by MarrowB83** [comment]
Yep, that is exactly the same behaviour that happend to me. :-) Thank you @jamadaha!

**Comment by Snufflenose** [comment]
I wanted to add two more issues regarding the Defias Traitor:

1. Buffing him enables PvP (I'm not sure this is supposed to happen).

2. After I had started the escort I met someone in Moonbrook who invited me into a group - at the very same moment the Defias Traitor disappeared (despawned?).
